integrations/apollo/src/index.ts

```diff
@@ -124,6 +133,20 @@ export const createApolloClient = ({
       )
     : httpLink
 
+  const links = [];
+
+  if(onError) {
+    links.push(onError);
+  }
+
+  if(customLink) {
+    links.push(customLink);
+  }
+
+  links.push(splitLink);
+
+  const link = from(generateLinks(links))
```

Comment:
Now that `generateLinks` is not mandatory, it's possible that this function is undefined. Could you please handle that case too?

Reply:
That's why I had it default to a function :) I'll remove that and check if it's set or not

Reply:
Oh, well. I didn't notice that, sorry. It's fine if you leave it like that.